### PR TITLE
Update to go 0.13 and remove deprecated dep managers

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We would like to take advantage of `0.13` features and whilst upgrading, remove the deprecated dep managers that currently sat in the container.

### Solution

* Update base golang image.
* Remove old dep managers.
* Update build script now dep mangers have gone.

### Notes

- **Remember** to tag your merge commit after merging 👍
- **Remember** to add the Clubhouse ticket url as a comment ♣️ 🏠

### CODE REVIEW

**Comment by stevenjack** [comment]
https://app.clubhouse.io/vidsy/story/12565/upgrade-go-to-0-13

File: Dockerfile [hunk 1]
```diff
@@ -13,35 +7,26 @@ RUN apk update \
...
-FROM golang:1.12.5-alpine3.9
+FROM golang:1.13.6-alpine3.11
 LABEL maintainer="Vidsy <tech@vidsy.co>"
 
 ARG VERSION
 LABEL version=$VERSION
 
 ENV CGO_ENABLED 0
 ENV GOCACHE /go/pkg/cache
+ENV GOPRIVATE github.com/vidsy/*
```
Comment (stevenjack) [inline Dockerfile:22]: @krak3n @luk4ward Thoughts on this being in here?

It is a public repo, so obviously this isn't relevant to anyone outside of the Vidsy org.

If not in here we could add it to a context in CircleCI but would have to make sure that context was always applied to each go job.

Comment (krak3n) [reply Dockerfile:22]: I'm not adverse to this being here, but I think I would prefer a CircleCI context.

**Comment by krak3n** [comment]
:cake:

**Review by krak3n — APPROVED** [review]

**Pull request merged**